watchlist.py

```diff
@@ -218,10 +195,12 @@ def requestWatchlist(tryAgain=True):
                 tokensFile.truncate()
 
             user = getUserForPlexToken(token['token'])

```

Comment:
Instead of this whole task we should probably just properly report on the error communicating with overseerr from directly within `getUserForPlexToken`.